hie-apply-refact/Haskell/Ide/ApplyRefactPlugin.hs

```diff
@@ -1,9 +1,10 @@
 {-# OPTIONS_GHC -fno-warn-partial-type-signatures #-}
 
-{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE CPP                   #-}
```

Comment:
I like having them sorted. Then my OCD bit kicked in and I aligned the closing.  I don't feel strongly about the endings, do about the sorting.  And I guess I need to start considering hindent.